migrations/v2.js

```diff
@@ -0,0 +1,131 @@
+    return true;
+  });
+  checkContent('adapt-contrib-textInput - check _canShowMarking added', async () => {
+    const isValid = components.every(component => _.has(component, '_canShowMarking'));
+    if (!isValid) throw new Error('_canShowMarking not added');
+    return true;
+  });
+  updatePlugin('adapt-contrib-textInput - update to v2.0.5', { name: 'adapt-contrib-textInput', version: '2.0.5', framework: '>=2.0.11' });
+});
+
+describe('adapt-contrib-textInput - v2.0.0 to v2.1.0', async () => {
+  let course;
+  const ariaRegionPath = '_globals._components._textInput.ariaRegion';
+  whereFromPlugin('adapt-contrib-textInput - from v2.0.0 to v2.1.0', { name: 'adapt-contrib-textInput', version: '<2.1.0' });
+  whereContent('adapt-contrib-textInput - where using old _globals ariaRegion default', async () => {
+    course = getCourse();
+    const oldAriaRegion = 'This question component requires you to input your answer in the textbox provided. When you have answered the question select the submit button below.';
+    return !_.has(course, ariaRegionPath) || _.get(course, ariaRegionPath) === oldAriaRegion;
+  });
+  const newAriaRegion = 'This question requires you to input your answer in the textbox provided. When you have done so, select the submit button below.';
+  mutateContent('adapt-contrib-textInput - update _globals ariaRegion default', async () => {
+    _.set(course, ariaRegionPath, newAriaRegion);
+    return true;
+  });
+  checkContent('adapt-contrib-textInput - check _globals ariaRegion default updated', async () => {
+    const isValid = _.get(course, ariaRegionPath) === newAriaRegion;
+    if (!isValid) throw new Error(`${ariaRegionPath} default not updated`);
+    return true;
+  });
+  updatePlugin('adapt-contrib-textInput - update to v2.1.0', { name: 'adapt-contrib-text', version: '2.1.0', framework: '>=2.0.11' });
+});
+
+describe('adapt-contrib-textInput - v2.0.0 to v2.2.0', async () => {
+  let components;
+  const feedbackTitlePath = '_feedback.title';
+  whereFromPlugin('adapt-contrib-textInput - from v2.0.0 to v2.2.0', { name: 'adapt-contrib-textInput', version: '<2.2.0' });
+  whereContent('adapt-contrib-textInput - where missing _feedback.title', async () => {
+    components = getComponents('textinput').filter(component => !_.has(component, feedbackTitlePath));
+    return Boolean(components.length);
+  });
+  mutateContent('adapt-contrib-textInput - add _feedback.title', async () => {
+    components.forEach(component => {
+      _.set(component, feedbackTitlePath, '');
+    });
+    return true;
+  });
+  checkContent('adapt-contrib-textInput - check _feedback.title added', async () => {
+    const isValid = components.every(component => _.has(component, feedbackTitlePath));
+    if (!isValid) throw new Error(`${feedbackTitlePath} not added`);
+    return true;
+  });
+  updatePlugin('adapt-contrib-textInput - update to v2.2.0', { name: 'adapt-contrib-textInput', version: '2.2.0', framework: '>=2.0.11' });
```

Comment:
A recent change to stop it from trying to migrate anything lower than v2.
```suggestion
  whereFromPlugin('adapt-contrib-textInput - from v2.0.0 to v2.0.2', { name: 'adapt-contrib-textInput', version: '>=2.0.0 <2.0.2' });
```